src/Nethermind/Nethermind.Evm/BlockExecutionContext.cs

```diff
@@ -3,32 +3,35 @@
 
 using System;
 using Nethermind.Core;
+using Nethermind.Core.Crypto;
 using Nethermind.Core.Specs;
 using Nethermind.Int256;
 
 namespace Nethermind.Evm;
 
-public readonly struct BlockExecutionContext
+public readonly struct BlockExecutionContext(BlockHeader blockHeader, in UInt256 blobBaseFee)
```

Comment:
Could go all out and add `[StructLayout(LayoutKind.Auto)]` though don't think will make much difference here as switches to auto if you have object references

Reply:
we can try in next pr